examples/common/rules.mk

```diff
@@ -27,13 +27,14 @@ XI_EXAMPLE_BIN := $(XI_EXAMPLE_BINDIR)/$(XI_EXAMPLE_NAME)
 
 XI_CLIENT_PATH ?= $(CURDIR)/../../
 XI_CLIENT_INC_PATH += $(CURDIR)/../../include
+XI_CLIENT_INC_PATH += $(CURDIR)/../../include/bsp
 XI_CLIENT_LIB_PATH ?= $(CURDIR)/../../bin/$(XI_TARGET_PLATFORM)
 
 XI_CLIENT_ROOTCA_LIST := $(CURDIR)/../../res/trusted_RootCA_certs/xi_RootCA_list.pem
 
 XI_FLAGS_INCLUDE += $(foreach i,$(XI_CLIENT_INC_PATH),-I$i)
 
-XI_FLAGS_COMPILER ?= -Wall -Werror -Wno-pointer-arith -Wno-format -fstrict-aliasing -Os -Wextra
+XI_FLAGS_COMPILER ?= -Wall -Werror -Wno-pointer-arith -Wno-format -fstrict-aliasing -Os -Wextra -DUSE_CBOR_CONTEXT
```

Comment:
OK, so this preoprocessor doesn't actually alter any function signatures.  The scary thing is that we're blindly adding it regardless of library configuration due to it currently being benign -- it only adds some preprocessor definitions but doesn't change signatures.  If it ever did change signatures then including it here but not including it in the library when built will cause unexpected crashes.  

That's why I ask.